### ISSUE TITLE
bixal-site-drupal - bug: Fix HTML in blog view

### ISSUE THREAD

**Issue by mejiaj** [Bixal/bixal-site-drupal#415]
**Describe the bug**
Blog teasers have malformed HTML.

**To Reproduce**
1. Go to https://www.bixal.com/blog
2. In devtools, inspect a blog teaser. The full class is `bix-blurb node node--type-blog node--view-mode-teaser`
3. Confirm that its parent is a `div`, not the `ul`.

Alternatively run axe a11y scan on this page.

**Expected behavior**
HTML should be valid and semantic.

**Screenshots**

![Image](https://github.com/user-attachments/assets/b11760d2-3b0f-4add-aa04-db06a32ad826)

![Image](https://github.com/user-attachments/assets/a99eb0a3-fd8e-4713-8e6b-666a128de133)
